.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature Request
+about: For Requesting enhancement or changes to API functionality
+
+---
+
+## APIs Affected
+
+<!-- Please list affected APIs -->
+
+* API 1
+* API 2
+
+## Explanation Feature
+
+<!-- One or two sentence summary of the Request. -->
+
+## Value Provided by Feature
```

Comment:
@ayaleloehr when you stated `value` above, did you mean the impact value? Or more specifically, how would this feature request will impact the APIs overall value to developers + the public?

Reply:
I'm flexible I guess.  When I wrote that, what I had in my head was someone answering: "If I use the API as it exists today it will be really hard to integrate with my application because of X, but if you made Y change/addition, I could integrate much more easily and build a better product using this API" or something.  But that doesn't have to be the be-all-end-all and open to better ideas.